statsbombpy/sb.py

```diff
@@ -213,7 +213,7 @@ def competition_frames(
         fmt="json",
         creds=creds,
     )
-    with Pool(PARALLELL_CALLS_NUM) as p:
+    with Pool(MAX_CONCURRENCY) as p:
```

Comment:
Do we know what happens if a user does something stupid like specifying more cores than they have? 

Might be useful to have a test for this.

Reply:
I will test this

Reply:
@jdragovichstatssomb it runs fine still but:

> If you specify more cores than your system has using the multiprocessing.Pool function in Python, then the function will still attempt to create that many worker processes. However, since there are not enough physical cores on your system, some of the worker processes will have to share a core with another process. This can lead to decreased performance and longer processing times.

Reply:
awesome. :shipit: